docs/installation.rst

```diff
@@ -0,0 +1,69 @@
+Installation
+============
+
+Graphene-Django takes a few seconds to install and set up.
+
+Requirements
+------------
+
+Graphene-Django currently supports the following versions of Django:
+
+* Django 2.0 >
+
+Installation
+------------
+
+.. code:: bash
+
+    pip install graphene-django
+
+**We strongly recommend pinning against a specific version of Graphene-Django as new versions could introduce breaking changes to your project.**
+
+Add ``graphene_django`` to the ``INSTALLED_APPS`` in the ``settings.py`` file of your Django project:
+
+.. code:: python
+
+    INSTALLED_APPS = [
+        ...
+        'django.contrib.staticfiles', # Required for GraphiQL
+        'graphene_django'
+    ]
+
+
+We need to add a graphql URL to the ``urls.py`` of your Django project:
+
+.. code:: python
+
+    from django.conf.urls import url
+    from graphene_django.views import GraphQLView
+
+    urlpatterns = [
+        # ...
+        url(r'^graphql', GraphQLView.as_view(graphiql=True)),
+    ]
+
+(Changed ``graphiql=True`` to ``graphiql=False`` if you do not want to use the graphiQL API browser.)
```

Comment:
"Changed" => "Change",
"graphiQL" => "GraphiQL"

Newcomers to GraphQL may not know what GraphiQL is. Might be worth linking to "GraphiQL" to it with a `target="_blank` attribute to https://github.com/graphql/graphiql